packages/node/test/integrations/http.test.ts

```diff
@@ -246,97 +246,198 @@ describe('tracing', () => {
+          );
+
+          createTransactionAndPutOnScope(hub);
+
+          const request = http.get(url);
+
+          expect(request.getHeader('sentry-trace')).toBeDefined();
+          expect(request.getHeader('baggage')).toBeDefined();
+        },
       );
 
-      const transaction = createTransactionAndPutOnScope(hub);
-      const spans = (transaction as unknown as Span).spanRecorder?.spans as Span[];
+      it.each([
+        ['http://dogs.are.great/api/v1/index/', []],
+        ['http://cats.are.great/api/v1/index/', [/\/v2/]],
+        ['http://cats.are.great/api/v1/index/', [/\/(v2|v3)/]],
+        ['http://cats.are.great/api/v1/index/', [/dogs\.are\.great/, /dogs\.are\.not\.great/]],
+        ['http://cats.are.great/api/v1/index/', [/https:/]],
+        ['http://cats.are.great/api/v1/index/', ['dogs.are.great']],
+        ['http://cats.are.great/api/v1/index/', ['/api/v2']],
+        ['http://cats.are.great/api/v1/index/', ['https://']],
+      ])(
+        'doesn\'t attach trace inforation to header of outgoing requests when url doesn\'t match tracePropagationTargets (url="%s", tracePropagationTargets=%p)',
+        (url, tracePropagationTargets) => {
+          nock(url).get(/.*/).reply(200);
+
+          const httpIntegration = new HttpIntegration({ tracing: true });
 
-      const request = http.get(url);
+          const hub = createHub({ tracePropagationTargets });
 
-      // There should be no http spans
-      const httpSpans = spans.filter(span => span.op?.startsWith('http'));
-      expect(httpSpans.length).toBe(0);
+          httpIntegration.setupOnce(
+            () => undefined,
+            () => hub,
+          );
 
-      // And headers are not attached without span creation
-      expect(request.getHeader('sentry-trace')).toBeUndefined();
-      expect(request.getHeader('baggage')).toBeUndefined();
+          createTransactionAndPutOnScope(hub);
+
+          const request = http.get(url);
+
+          expect(request.getHeader('sentry-trace')).not.toBeDefined();
+          expect(request.getHeader('baggage')).not.toBeDefined();
+        },
+      );
     });
 
-    it.each([
-      ['http://dogs.are.great/api/v1/index/', [/.*/]],
-      ['http://dogs.are.great/api/v1/index/', [/\/api/]],
-      ['http://dogs.are.great/api/v1/index/', [/\/(v1|v2)/]],
-      ['http://dogs.are.great/api/v1/index/', [/dogs\.are\.great/, /dogs\.are\.not\.great/]],
-      ['http://dogs.are.great/api/v1/index/', [/http:/]],
-      ['http://dogs.are.great/api/v1/index/', ['dogs.are.great']],
-      ['http://dogs.are.great/api/v1/index/', ['/api/v1']],
-      ['http://dogs.are.great/api/v1/index/', ['http://']],
-      ['http://dogs.are.great/api/v1/index/', ['']],
-    ])(
-      'attaches trace inforation to header of outgoing requests when url matches tracePropagationTargets (url="%s", tracePropagationTargets=%p)',
-      (url, tracePropagationTargets) => {
+    describe('as BrowserTracing constructor options', () => {
```

Comment:
l: I think we should rename this, right?

```suggestion
    describe('as Http integration constructor options', () => {
```